### PR TITLE
[DOCS] Update Add Validation Data or Expectation Suites to a Checkpoint

### PR DESCRIPTION
**Pull request by kwcanuck**
In [DSB-408](https://greatexpectations.atlassian.net/browse/DSB-408), @donaldheppner requested that the topic _Add validation data or Expectation suites to a Checkpoint_ be updated "... to reflect both YAML and Python-style configuration and brought under test". This PR implements these updates. 

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] _Appropriate_ tests and docs have been updated

[DSB-408]: https://greatexpectations.atlassian.net/browse/DSB-408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md [hunk 1]
```diff
@@ -17,146 +17,20 @@ Add validation data or <TechnicalTag tag="expectation_suite" text="Expectation S
...
-    runtime_configuration:
-      result_format:
-        result_format: BASIC
-        partial_unexpected_count: 20
-```
-
 ## Add an Expectation Suite to the Checkpoint
 
-To add a second Expectation Suite (in this example we add ``users.error``) to your Checkpoint configuration, modify the file to add an additional `batch_request` key and corresponding information, including `evaluation_parameters`, `action_list`, `runtime_configuration`, and `expectation_suite_name`.  In fact, the simplest way to run a different Expectation Suite on the same <TechnicalTag tag="batch" text="Batch" /> of data is to make a copy of the original `batch_request` entry and then edit the `expectation_suite_name` value to correspond to a different Expectation Suite.  The resulting configuration will look like this:
+To add a second Expectation Suite (in this example we add ``users.error``) to your Checkpoint configuration, update the validations in your Checkpoint.  The resulting configuration will look like this:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md]:
```suggestion
To add a second Expectation Suite (in the following example, `users.error`) to your Checkpoint configuration, you update the Validations in your Checkpoint.  The configuration appears similar to this example:
```

File: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md [hunk 2]
```diff
@@ -17,146 +17,20 @@ Add validation data or <TechnicalTag tag="expectation_suite" text="Expectation S
...
 ```
 
 ## Add validation data to the Checkpoint
 
-In the prvious example, the entry you added with your Expectation Suite was paired with the same Batch of data as the original Expectation Suite.  However, you may also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> (and thus different Batches of data) when you add an Expectation Suite.  The flexibility of easily adding multiple Validations of Batches of data with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> can be demonstrated using the following example of a Checkpoint configuration file:
+In the previous example, the validation you added with your Expectation Suite was paired with the same Batch of data as the original Expectation Suite.  However, you may also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> (and thus different Batches of data) when you add an Expectation Suite.  The flexibility of easily adding multiple Validations of Batches of data with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> can be demonstrated using the following example:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md]:
```suggestion
In the previous example, the Validation you added was paired with the same Batch as the original Expectation Suite.  However, you can also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> when you add an Expectation Suite.  Adding multiple Validations with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> is demonstrated in the following example:
```

File: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md [hunk 3]
```diff
@@ -17,146 +17,20 @@ Add validation data or <TechnicalTag tag="expectation_suite" text="Expectation S
...
+According to this Checkpoint configuration, the Expectation Suite ``users.warning`` is run against the ``batch_request`` with the results processed by the Actions specified in the top-level ``action_list``. Similarly, the Expectation Suite ``users.error`` is run against the ``batch_request`` with the results also processed by the actions specified in the top-level ``action_list``. In addition, the top-level Expectation Suite ``users.delivery`` is run against the ``batch_request`` with the results processed by the union of actions in the validations ``action_list`` and in the top-level ``action_list``.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md]:
```suggestion
In this Checkpoint configuration, the Expectation Suite `users.warning` runs against the `batch_request` with the results processed by the Actions specified in the `action_list`. Similarly, the Expectation Suite `users.error` runs against the `batch_request` with the results processed by the actions specified in the `action_list`. In addition, the Expectation Suite `users.delivery` runs against the `batch_request` with the results processed by actions in the Validation `action_list` and the `action_list`.
```

File: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md [hunk 4]
```diff
@@ -17,146 +17,20 @@ Add validation data or <TechnicalTag tag="expectation_suite" text="Expectation S
...
-      param1: "$MY_PARAM"
-      param2: 1 + "$OLD_PARAM"
-    runtime_configuration:
-      result_format:
-        result_format: BASIC
-        partial_unexpected_count: 20
+```python name="tests/integration/docusaurus/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.py add_expectation_suite"
 ```
 
 ## Add validation data to the Checkpoint
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md]:
```suggestion
## Add Validation data to the Checkpoint
```

File: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md [hunk 5]
```diff
@@ -17,146 +17,20 @@ Add validation data or <TechnicalTag tag="expectation_suite" text="Expectation S
...
 
-In the prvious example, the entry you added with your Expectation Suite was paired with the same Batch of data as the original Expectation Suite.  However, you may also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> (and thus different Batches of data) when you add an Expectation Suite.  The flexibility of easily adding multiple Validations of Batches of data with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> can be demonstrated using the following example of a Checkpoint configuration file:
+In the previous example, the Validation you added was paired with the same Batch as the original Expectation Suite.  However, you can also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> when you add an Expectation Suite.  Adding multiple Validations with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> is demonstrated in the following example:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md]:
```suggestion
In the previous example, you added a Validation and it was paired with the same Batch as the original Expectation Suite.  However, you can also specify different <TechnicalTag tag="batch_request" text="Batch Requests" /> when you add an Expectation Suite.  Adding multiple Validations with different Expectation Suites and specific <TechnicalTag tag="action" text="Actions" /> is shown in the following example:
```

File: docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md [hunk 6]
```diff
@@ -17,146 +17,20 @@ Add validation data or <TechnicalTag tag="expectation_suite" text="Expectation S
...
+In this Checkpoint configuration, the Expectation Suite `users.warning` runs against the `batch_request` with the results processed by the Actions specified in the `action_list`. Similarly, the Expectation Suite `users.error` runs against the `batch_request` with the results processed by the actions specified in the `action_list`. In addition, the Expectation Suite `users.delivery` runs against the `batch_request` with the results processed by actions in the Validation `action_list` and the `action_list`.
```
Comment (kwcanuck) [inline docs/docusaurus/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md]:
```suggestion
In this Checkpoint configuration, the Expectation Suite `users.warning` runs against the `batch_request` and the results are processed by the Actions specified in the `action_list`. Similarly, the Expectation Suite `users.error` runs against the `batch_request` and the results processed by the actions specified in the `action_list`. In addition, the Expectation Suite `users.delivery` runs against the `batch_request` and the results are processed by the actions in the Validation `action_list` and its `action_list`.
```

**Review by Rachel-Reverie — APPROVED** [review]
One thing: the PR states that it should include both YAML and python methods for completing the task -- I'm presuming the original description of the PR is outdated, as we've been moving away from YAML based guides (and I didn't see any YAML covered in the updated document or test script).  As long as that is the case, then this LGTM.

**Comment by anthonyburdi** [comment]
Force merged after all CI checks passed to enable releasing today.

**Pull request merged**